install_config/registry/extended_registry_configuration.adoc

```diff
@@ -436,6 +436,61 @@ more efficient registry operation. On the other hand, memory usage will raise
 as well as a risk of serving image layer to user, who is no longer authorized
 to access it.
 
+[[s3-driver]]
+==== S3
+The main documentation about S3 driver configuration is
+link:https://docs.docker.com/registry/storage-drivers/s3/[here].
+
+If you want to use a S3 region that is not supported by the integrated
+registry you are using, then you can specify a `regionendpoint` to avoid the
+`region` validation error.
+
+For more information about the regions and endpoints for Amazon S3, see
+link:https://docs.aws.amazon.com/general/latest/gr/rande.html#s3_region[Regions
+and Endpoints].
+
+For example:
+[source,yaml]
+----
+version: 0.1
+log:
+  level: debug
+http:
+  addr: :5000
+storage:
+  cache:
+    blobdescriptor: inmemory
+  delete:
+    enabled: true
+  s3:
+    accesskey: BJKMSZBRESWJQXRWMAEQ
+    secretkey: 5ah5I91SNXbeoUXXDasFtadRqOdy62JzlnOW1goS
+    bucket: docker.myregistry.com
+    region: eu-west-3
+    regionendpoint: https://s3.eu-west-3.amazonaws.com
+ auth:
+  openshift:
+    realm: openshift
+middleware:
+  registry:
+    - name: openshift
+  repository:
+    - name: openshift
+  storage:
+    - name: openshift
+----
+
+[NOTE]
+====
+It is very important in this case that the `region` and `regionendpoint` fields
+are consistent between themselves. Otherwise the integrated registry will start,
+but it can not read or write anything to the S3 storage.
+====
+
+The `regionendpoint` can also be useful if you use a different from the
```

Comment:
"if you use a different from" - there's a word missing here but i'm not sure what it is.

Reply:
Fixed.